.github/workflows/ReceivePR.yml

```diff
@@ -0,0 +1,39 @@
+name: Receive PR
+
+on:
+  pull_request:
+
```

Comment:
Shall we add a `path-ignore:` filter for workflows and docs?

Reply:
I can do it. We will have to bypass the branch protection then.

Reply:
ok then let's not.

Reply:
I don't have any issues with that. These kind of PRs are not common and don't have any impact on the tests.